WooCommerce/Classes/POS/Utils/StockFormatter.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import struct Yosemite.POSSimpleProduct
+
+struct StockFormatter {
```

Comment:
nit: how about a prefix `POS`? Also, does it make sense to have this in Yosemite?

Reply:
> how about a prefix POS? 

Sounds good! https://github.com/woocommerce/woocommerce-ios/pull/15670/commits/8a5ec53b4d60bcd5d051fd0f58a548c89935ee0c

> Also, does it make sense to have this in Yosemite?

I'm not sure, but at the moment I don't think it does as we're just dealing with the outer layer where the UI and business logic intersect. I'll keep this in mind thought as we move forward as might become more Yosemite-like.